src/test/java/uk/gov/pay/api/validation/PaymentRequestValidatorTest.java

```diff
@@ -13,34 +13,80 @@
 
     @Rule
     public ExpectedException expectedException = ExpectedException.none();
-    private static final String SUCCESSFUL_TEST_EMAIL = "alice.111@mail.fake";
-    private static final String UNSUCCESSFUL_TEST_EMAIL = randomAlphanumeric(255) + "@mail.fake";
+
+    private static final int VALID_AMOUNT = 100;
+    private static final String VALID_RETURN_URL = "https://www.example.com/return_url";
+    private static final String VALID_REFERENCE = "reference";
+    private static final String VALID_DESCRIPTION = "description";
+
+    private final PaymentRequestValidator paymentRequestValidator = new PaymentRequestValidator(URLValidator.SECURITY_ENABLED);
+
+    @Test
+    public void validParameters_shouldSuccessValue() {
+        CreatePaymentRequest createPaymentRequest = createPaymentRequest(VALID_AMOUNT, VALID_RETURN_URL, VALID_REFERENCE, VALID_DESCRIPTION);
+        paymentRequestValidator.validate(createPaymentRequest);
+    }
+
+    @Test
+    public void validateMinimumAmount_shouldSuccessValue() {
+        CreatePaymentRequest createPaymentRequest = createPaymentRequest(PaymentRequestValidator.AMOUNT_MIN_VALUE, VALID_RETURN_URL, VALID_REFERENCE, VALID_DESCRIPTION);
+        paymentRequestValidator.validate(createPaymentRequest);
+    }
 
     @Test
-    public void validateMinimumAmount_shouldSuccessValue(){
-        PaymentRequestValidator paymentRequestValidator = new PaymentRequestValidator(URLValidator.SECURITY_ENABLED);
-        CreatePaymentRequest paymentRequest = new CreatePaymentRequest(1, "https://return.to/return_url", "reference", "description");
+    public void validateMinimumAmount_shouldFailValue() throws Exception {
```

Comment:
I assume what this test tests for has been intentionally changed because it wrong previously. (Before it claimed to be checking going below the minimum amount causes a validation failure but it was actually checking going above the maximum amount causes a validation failure.)